specification/recoveryservices/Recoveryservices.Management/VaultExtendedInfoResource.tsp

```diff
@@ -0,0 +1,77 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+import "./Vault.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.RecoveryServices;
+/**
+ * Vault extended information.
+ */
+@singleton("vaultExtendedInfo")
+@parentResource(Vault)
+model VaultExtendedInfoResource
```

Comment:
>VaultExtendedInfoResource [](http://example.com/codeflow?start=6&length=25)

do we want Resource in the name, really?

Reply:
We just follow what it defined before. They have `Resource` before.